frontend/src/routes/SearchPage/__snapshots__/searchDefinitions.test.ts.snap

```diff
@@ -1421,9 +1421,10 @@ Array [
     "sort": "name",
   },
   Object {
-    "cell": "namespace",
-    "header": "Namespace",
-    "sort": "namespace",
+    "cell": "scope",
+    "header": "Scope",
+    "sort": "scope",
+    "tooltip": "Scope refers to the cluster that is affected by the PolicyReport.",
```

Comment:
Would `associated` be more appropriate than affected?

Reply:
`Scope refers to the cluster associated to the PolicyReport.`?

Reply:
Yea, what do you think about that update? 

I suggest `associated` bc the cluster the violation is listed within PolicyReport vs the PolicyReport causing the violations within the cluster

Reply:
Works for me - i'll make the change. Thanks!